openapi/jira/Ballerina.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Jira", "Projects", "Issues"]
+org = "ballerinax"
+name = "jira"
+version = "0.1.1-SNAPSHOT"
```

Comment:
Can you change the version to `0.1.0-SNAPSHOT` instead of `0.1.1-SNAPSHOT`?

Reply:
Yes. Noted